README.md

```diff
@@ -9,16 +9,38 @@
 
 ## Overview
 
-HealthyHome is an open-source smart home device which monitors the indoors environment conditions, based on a Raspberry Pi.
+HealthyHome is an open-source project centred around a smart home device which monitors the indoor environment conditions of a home.
+
+### Objectives
+
+-   Allow a user to characterise important parameters of their indoor home environment
+-   Provide easy access to both real-time and historical versions of this information
+-   Allow interested parties to recreate this project
+
+### Features
+
+-   Monitors the ambient temperature, barometric pressure, relative humidity, UV index, ambient light levels and concentration of particles in the air
```

Comment:
Highlight the most important features, and keep an order from most to least important.

Reply:
> > > > > @roryort you could add an **How-to** section where you say that instructions on how to reproduce the project will be given as development goes on. You can also give some sort of primer on the project Current status? Not sure... perhaps there is some cool way to do this, like with shields
> > > > 
> > > > 
> > > > I think we should create a separate file containing the **How To** instructions. Having a full list of instructions on the readme may convolute the readme. We can link that file to this readme. Thoughts?
> > > 
> > > 
> > > READMEs do contain a Quick start section, so it'd be this. If instructions do get long, the section can still link for an in-depth instructions page
> > 
> > 
> > I can create a **Quick Start** section which provides instructions on how to reproduce the project and leave space for a **How To** document for in-depth instructions
> 
> My idea was that Quick Start would be replacing How-to altogether

Ok

Reply:
Done